README.md

```diff
@@ -39,6 +39,14 @@ one to understand and see the effect). As soon as you save a change, the app
 should refresh in the browser with the new content. Our intent is that this time
 is less than two seconds, even for a large application.
 
+## Production bundling with Rollup
+
+```
+npm run rollup
+```
+
+Test the bundled rollup application by running `npm run lite-server`
```

Comment:
I would also include the Bazel versions of the commands in this section, not just the `npm run` aliases. The first time I read this, I was confused because I assumed these `npm run` commands were circumventing Bazel entirely. Only when I saw package.json did I realize the npm commands were aliases to Bazel commands.

Reply:
Sure. I'll just change the README to show to bazel build commands